bootstrap/apollo-env.sh

```diff
@@ -14,3 +16,10 @@ if [[ -z "${color_start-}" ]]; then
   declare -r color_norm="${color_start}0m"
 fi
 
```

Comment:
That sounds sensible, I didn't want to overwrite any previously/externally defined logging as you're quite often only interested in a particular application.